tensorflow/tools/pip_package/build_pip_package.sh

```diff
@@ -262,6 +262,13 @@ function main() {
     elif [[ "$1" == "--nightly_flag" ]]; then
       NIGHTLY_BUILD=1
     elif [[ "$1" == "--gpu" ]]; then
+      # Check that --cpu has not been passed.
```

Comment:
Instead of this, I recommend having a variable for GPU_BUILD, CPU_BUILD, and in addition GPUDIRECT_BUILD ROCM_BUILD.
Then, after all the flags are parsed, we make sure only one of these are set to 1.
We can do that by simply adding all of these, and checking if the total is less than or equal to 1.

Reply:
Done.